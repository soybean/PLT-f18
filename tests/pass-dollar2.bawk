BEGIN {}

LOOP {
	string hello;
	hello = $0;
	print(hello);
}

END {}
