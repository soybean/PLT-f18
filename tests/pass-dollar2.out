Hello
Line 2
This is the best line
Wow
Here we go

